whylog/tests/tests_client/__init__.py

```diff
@@ -1,9 +1,10 @@
 from unittest import TestCase
-from os.path import join as path_join
+from os.path import join as path_join, getsize as get_file_size
```

Comment:
please use `import os.path` and then use `os.path.join` etc. I know your way may be faster, but it is much harder to read.